File: server/service.py
from bankjson import JSONSerializable


class Service(JSONSerializable):
    def __init__(self, name, tokens, estimate, description):
        self.name = name
        self.tokens = tokens
        self.estimate = estimate
        self.description = description

    def to_json(self):
        return {'name': self.name,
            'tokens': self.tokens,
            'estimate': self.estimate,
            'description': self.description,
            }

File: android/app/src/main/java/com/whitespace/magicznybank/Service.java
package com.whitespace.magicznybank;

import java.util.Date;

/**
 * Created by Jakub on 19.09.2015.
 */
public class Service {
    public String id;
    public int tokens;
    public double estimate;
    public String name;
    public String description;
    public int price;

    public Service(String id, int tokens, double estimate, String name, String description, int price) {
        this.id = id;
        this.tokens = tokens;
        this.estimate = estimate;
        this.name = name;
        this.description = description;
        this.price = price;
    }

    public Service(String id, int tokens, double estimate, String name, String description) {
        this(id, tokens, estimate, name, description, 0);
    }
}
